src/isar/config/settings.env

```diff
@@ -0,0 +1,9 @@
+ISAR_ROBOT_PACKAGE = isar_turtlebot
+
```

Comment:
Shouldn't this be defaulted to 'isar_robot' ?

Reply:
Correct, good catch!